test/content-type-information-repository.test.js

```diff
@@ -10,7 +12,7 @@ const ContentTypeCache = require('../src/content-type-cache');
 const ContentTypeInformationRepository = require('../src/content-type-information-repository');
 const User = require('../src/user');
 const TranslationService = require('../src/translation-service');
-const H5pError = require('../src/helpers/h5p-error');
+const H5pError = require('../src/helpers/h5p-error');// eslint-disable-line no-unused-vars
```

Comment:
Why did you disable the linter check here?

Reply:
The original reason was that ESLint complained about required classes that were only imported to get better code completion in my IDE but not actually referenced in running code. Looks like H5pError isn't in use here, though.